lib/src/router/interfaces.dart

```diff
@@ -16,8 +16,16 @@ import 'instruction.dart' show ComponentInstruction;
 ///
 /// ### Example
 /// ```dart
-/// // {@disabled-source "router/ts/on_activate/on_activate_example.ts" region="routerOnActivate"}
+/// // {@source "docs/router/lib/crisis_center/crisis_detail_component.dart" region="routerOnActivate"}
+/// @override
+/// void routerOnActivate(next, prev) {
+///   print('Activating ${next.routeName} ${next.urlPath}');
+/// }
 /// ```
+///
+/// See the [router documentation][router] for details.
+///
+/// [router]: docs/guide/router/5#onactivate
```

Comment:
Agreed. Created https://github.com/dart-lang/site-webdev/issues/534 to track this.